advanced_functionality/mxnet_mnist_byom/mxnet_mnist.ipynb

```diff
@@ -58,7 +58,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
```

Comment:
At the top of the "Introduction" markdown can you add a line like:  "_This notebook was created and tested on an ml.p2.xlarge notebook instance._"